site/content/en/docs/manual/advanced/export-import.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Export/import a task'
+linkTitle: 'Export/import'
+weight: 25
+---
+
+In CVAT you can export and import tasks.
+This can be used to backup the task on your PC or to transfer the task to another server.
+
+## Export task
+
+To export the task, open the action menu and select `Export Task`.
+
+![](/images/image219_mapillary_vistas.jpg)
+
+As a result, you'll get a zip archive containing data, task specification and annotations with the following structure:
+
+```
+.
+├── data
+│   ├── {user uploaded data}
+│   ├── manifest.jsonl
+├── task.json
+└── annotations.json
+```
+
+Export task API:
+
+- endpoint: `/api/v1/tasks/{id}?action=export​`
+- method: `GET`
+- returns: zip archive
```

Comment:
@azhavoro 

Probably it returns 201, then 200, and finally 202 codes. Isn't it?
I mean that result is not always zip archive. Should we specify it here?